src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -447,6 +446,7 @@ def __call__(
         self,
         hidden_states,
         attention_mask,
+        indices_prng_key,
```

Comment:
Let's not pass the PRNG key as an argument to the self attention layer. Instead, what we can do is copy the logic that we employ for dropout and 'make' an RNG variable:
https://github.com/huggingface/transformers/blob/b210c83a78022226ce48402cd67d8c8da7afbd8d/src/transformers/models/big_bird/modeling_flax_big_bird.py#L388-L390

For the indices PRNG key, this will look something as follows:
```python
 indices_prng_key = None 
 if not deterministic: 
     indices_prng_key = self.make_rng("indices")
``` 

We can then forward the variable `indices_prng_key` to the relevant sparse attention functions.